packages/loot-core/src/client/actions/budgets.ts

```diff
@@ -209,9 +218,15 @@ export function downloadBudget(cloudFileId: string, { replace = false } = {}) {
         dispatch(setAppState({ loadingText: null }));
       } else if (error.reason === 'file-exists') {
         alert(
-          `A file with id “${error.meta.id}” already exists with the name “${error.meta.name}.” ` +
-            'This file will be replaced. This probably happened because files were manually ' +
-            'moved around outside of Actual.',
+          i18n.t(
+            'A file with id “{{id}}” already exists with the name “{{name}}”. ' +
+              'This file will be replaced. This probably happened because files were manually ' +
+              'moved around outside of Actual.',
+            {
+              id: error.meta.id,
+              name: error.meta.id,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏
```suggestion
              name: error.meta.name,
```